doc/migrations/3.3.0_3.3.0plus.md

```diff
@@ -47,8 +47,51 @@ You'll need to iterate over the whole `mam_muc_message` table with the following
 
 ### Cassandra
 
-TODO
+#### Step 1
+
+Please execute the following CQL statements on your MIM database:
+
+```
+USE mongooseim;
+ALTER TABLE mam_muc_message ADD from_jid varchar;
+CREATE INDEX ON mam_muc_message (from_jid);
+DESC mam_muc_message;
+```
+
+#### Step 2
+
+Now you have a schema that is compatible with MIM 3.3.0++ but isn't GDPR-compliant yet because the new column has no meaningful data.
+
+Please pick your favourite scripting/programming language and populate the new column with the help of a [dedicated script](jid-from-mam-muc-script.md).
+You'll need to iterate over the whole `mam_muc_message` table with the following algorithm:
+
+1. extract the whole `mam_muc_message` table:
+```
+SELECT * FROM mam_muc_message;
```

Comment:
Pretty much impossible for large DBs. Maybe it's better to recommend retrieving data by `id` in slices?

Reply:
yes, you right.

Reply:
well,  to filter by `id` we have to specify the key (some `room_jid` and `with_nick = ''`) or specify `ALLOW FILTERING`:
```
SELECT * FROM mongooseim.mam_muc_message LIMIT 10;
SELECT * FROM mongooseim.mam_muc_message where id > 399582468742569729 LIMIT 10  ALLOW FILTERING;
```

i believe `ALLOW FILTERING` is not the best idea, nor the room by room update.

cassandra drivers allow paging, i think this one should be used instead. but it's very driver specific to describe it here and CQL command in this case is still the same.